docs/acr42/03_system_scope_and_context.md

```diff
@@ -0,0 +1,63 @@
+# System Scope and Context
+
+![systemScopeAndContextBusiness](http://www.plantuml.com/plantuml/proxy?src=https://raw.githubusercontent.com/GenSpectrum/LAPIS/master/docs/arc42/images/systemScopeAndContext.puml)
+
+**LAPIS**
+- is provisioned by the maintainer, who provides a configuration file
+- can be accessed by the end-user through a REST interface with endpoints
+  - aggregated data
+    - returns how many sequences fulfill the provided filter criteria 
+  - aa-mutations, nuc-mutations
+    - returns a list of mutations (with proportion, count), which fulfill the provided filter criteria
+    - the proportion is relative to all sequences, which fulfill the provided criteria 
+  - details
+    - returns a list of metadata, for sequences which fulfill the provided criteria
+  - aa-sequence, nuc-sequence, nuc-sequence-aligned
+    - returns the corresponding sequences, which fulfill the provided criteria
+  - the provided criteria can be a compilation of
+    - metadata
+    - specific mutations
+    - pango lineage
+- the maintainer can trigger an update upon which LAPIS reads data from the disc and stores it internally 
+
+
+
+**Sequence data on disk**
+
+Preprocessing:
+- original data most likely provided by GISAID/Genbank
+- original data must be preprocessed (by for example Nextstrain), which includes the following steps 
+    - align nucleotide data
+    - assign amino acids
+    - determine insertions
+
+Required files:
+
+nucleotide_sequences.fasta
+- list of nucleotide sequences in the [fasta format](https://en.wikipedia.org/wiki/FASTA_format)
+
+nucleotide_sequences_aligned.fasta
+- same as nucleotide_sequences.fasta but aligned to reference genome
+
+aminoacid_sequences.fasta (multiple)
+- list of AA sequences in the fasta format
+- aligned
+- one file for each gene
+
+metadata_and_quality_control.tsv
+- list of metadata and quality control
+
+insertions.tsv
+- list of nucleotide insertions and AA insertions for each sequence
+
+alias.tsv
+- list of pangolin aliases
```

Comment:
Do we want to keep this here? The system needs to be reloaded, when the aliases change in this case. But is this more likely than that new data will arrive?